### ISSUE TITLE
PPA version doesn't start on ubuntu 22.10 (kinetic)

### ISSUE THREAD

**Issue by piontec** [atareao/my-weather-indicator#111]
Launching from terminal only gives:

```
/opt/extras.ubuntu.com/my-weather-indicator/bin/my-weather-indicator -v    
Namespace GeocodeGlib not available for version 1.0
Repository version required not present
```

It seems ubuntu only ships v2 of that lib:

```
apt search GeocodeGlib            
Sorting... Done
Full Text Search... Done
gir1.2-geocodeglib-2.0/kinetic,now 3.26.3-3 amd64 [installed]
  introspection data for geocode-glib library
```

**Comment by atareao** [comment]
I don't understand anything.... I removed this dependency 😔

What version of my-weather-indicator did you installed?

Thanks

**Comment by tuxaddict** [comment]
As shown in [comment](https://github.com/atareao/my-weather-indicator/issues/106#issuecomment-1509626855), im facing the exact same situation. i just tested this a minute ago again, same result.

As an additional information: Ubuntu (Unity) 23.04 shows the same behavior.

> Namespace GeocodeGlib not available for version 1.0
Repository version required not present

Since it does not start, im not sure how to figure out the version, sorry.

**Comment by tuxaddict** [comment]
> I don't understand anything.... I removed this dependency pensive
> 
> What version of my-weather-indicator did you installed?
> 
> Thanks

maybe this information could help?

> tux@box:my-weather-indicator$ sudo apt install my-weather-indicator 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  gir1.2-appindicator3-0.1 python3-pytz-deprecation-shim python3-requests-oauthlib python3-tzlocal
The following NEW packages will be installed:
  gir1.2-appindicator3-0.1 my-weather-indicator python3-pytz-deprecation-shim python3-requests-oauthlib python3-tzlocal
0 upgraded, 5 newly installed, 0 to remove and 3 not upgraded.
Need to get 4.375 kB of archives.
After this operation, 18,9 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 gir1.2-appindicator3-0.1 amd64 12.10.1+20.10.20200706.1-0ubuntu1 [3.452 B]
Get:2 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 python3-pytz-deprecation-shim all 0.1.0.post0-2 [14,4 kB]
Get:3 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 python3-requests-oauthlib all 1.3.0+ds-0.1 [18,7 kB]
Get:4 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 python3-tzlocal all 4.2-1 [21,5 kB]
Get:5 https://ppa.launchpadcontent.net/atareao/atareao/ubuntu kinetic/main amd64 my-weather-indicator all 0.10.1-0extras23.04.00 [4.317 kB]
Fetched 4.375 kB in 1s (4.770 kB/s)                                                 
Selecting previously unselected package gir1.2-appindicator3-0.1.
(Reading database ... 329389 files and directories currently installed.)
Preparing to unpack .../gir1.2-appindicator3-0.1_12.10.1+20.10.20200706.1-0ubuntu1_amd64.deb ...
Unpacking gir1.2-appindicator3-0.1 (12.10.1+20.10.20200706.1-0ubuntu1) ...
Selecting previously unselected package python3-pytz-deprecation-shim.
Preparing to unpack .../python3-pytz-deprecation-shim_0.1.0.post0-2_all.deb ...
Unpacking python3-pytz-deprecation-shim (0.1.0.post0-2) ...
Selecting previously unselected package python3-requests-oauthlib.
Preparing to unpack .../python3-requests-oauthlib_1.3.0+ds-0.1_all.deb ...
Unpacking python3-requests-oauthlib (1.3.0+ds-0.1) ...
Selecting previously unselected package python3-tzlocal.
Preparing to unpack .../python3-tzlocal_4.2-1_all.deb ...
Unpacking python3-tzlocal (4.2-1) ...
Selecting previously unselected package my-weather-indicator.
Preparing to unpack .../my-weather-indicator_0.10.1-0extras23.04.00_all.deb ...
Unpacking my-weather-indicator (0.10.1-0extras23.04.00) ...
Setting up python3-pytz-deprecation-shim (0.1.0.post0-2) ...
Setting up gir1.2-appindicator3-0.1 (12.10.1+20.10.20200706.1-0ubuntu1) ...
Setting up python3-tzlocal (4.2-1) ...
Setting up python3-requests-oauthlib (1.3.0+ds-0.1) ...
Setting up my-weather-indicator (0.10.1-0extras23.04.00) ...
Processing triggers for fontconfig (2.13.1-4.4ubuntu1) ...
Processing triggers for bamfdaemon (0.5.6+22.04.20220217-0ubuntu1) ...
Rebuilding /usr/share/applications/bamf-2.index...
Processing triggers for desktop-file-utils (0.26-1ubuntu4) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
tux@box:my-weather-indicator$ /opt/extras.ubuntu.com/my-weather-indicator/bin/my-weather-indicator --version
Namespace GeocodeGlib not available for version 1.0
Repository version required not present
me@pro:my-weather-indicator$ 

ps, as mentioned, same behavior on 23.04. thanks for your time and help (-:

**Comment by tuxaddict** [comment]
**YES!!! THAT WORKS!!!**

`ps³: an grep -ir 'geocodeglib' my-weather-indicator finds:
my-weather-indicator/share/my-weather-indicator/myweatherindicator.py:    gi.require_version('GeocodeGlib', '1.0')
so let me see if this is manipulatable (-:`

**Comment by atareao** [comment]
> > I don't understand anything.... I removed this dependency pensive
> > 
> > What version of my-weather-indicator did you installed?
> > 
> > Thanks
> 
> maybe this information could help?
> 
> > tux@box:my-weather-indicator$ sudo apt install my-weather-indicator 
> Reading package lists... Done
> Building dependency tree... Done
> Reading state information... Done
> The following additional packages will be installed:
>   gir1.2-appindicator3-0.1 python3-pytz-deprecation-shim python3-requests-oauthlib python3-tzlocal
> The following NEW packages will be installed:
>   gir1.2-appindicator3-0.1 my-weather-indicator python3-pytz-deprecation-shim python3-requests-oauthlib python3-tzlocal
> 0 upgraded, 5 newly installed, 0 to remove and 3 not upgraded.
> Need to get 4.375 kB of archives.
> After this operation, 18,9 MB of additional disk space will be used.
> Do you want to continue? [Y/n] y
> Get:1 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 gir1.2-appindicator3-0.1 amd64 12.10.1+20.10.20200706.1-0ubuntu1 [3.452 B]
> Get:2 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 python3-pytz-deprecation-shim all 0.1.0.post0-2 [14,4 kB]
> Get:3 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 python3-requests-oauthlib all 1.3.0+ds-0.1 [18,7 kB]
> Get:4 http://de.archive.ubuntu.com/ubuntu kinetic/universe amd64 python3-tzlocal all 4.2-1 [21,5 kB]
> Get:5 https://ppa.launchpadcontent.net/atareao/atareao/ubuntu kinetic/main amd64 my-weather-indicator all 0.10.1-0extras23.04.00 [4.317 kB]
> Fetched 4.375 kB in 1s (4.770 kB/s)                                                 
> Selecting previously unselected package gir1.2-appindicator3-0.1.
> (Reading database ... 329389 files and directories currently installed.)
> Preparing to unpack .../gir1.2-appindicator3-0.1_12.10.1+20.10.20200706.1-0ubuntu1_amd64.deb ...
> Unpacking gir1.2-appindicator3-0.1 (12.10.1+20.10.20200706.1-0ubuntu1) ...
> Selecting previously unselected package python3-pytz-deprecation-shim.
> Preparing to unpack .../python3-pytz-deprecation-shim_0.1.0.post0-2_all.deb ...
> Unpacking python3-pytz-deprecation-shim (0.1.0.post0-2) ...
> Selecting previously unselected package python3-requests-oauthlib.
> Preparing to unpack .../python3-requests-oauthlib_1.3.0+ds-0.1_all.deb ...
> Unpacking python3-requests-oauthlib (1.3.0+ds-0.1) ...
> Selecting previously unselected package python3-tzlocal.
> Preparing to unpack .../python3-tzlocal_4.2-1_all.deb ...
> Unpacking python3-tzlocal (4.2-1) ...
> Selecting previously unselected package my-weather-indicator.
> Preparing to unpack .../my-weather-indicator_0.10.1-0extras23.04.00_all.deb ...
> Unpacking my-weather-indicator (0.10.1-0extras23.04.00) ...
> Setting up python3-pytz-deprecation-shim (0.1.0.post0-2) ...
> Setting up gir1.2-appindicator3-0.1 (12.10.1+20.10.20200706.1-0ubuntu1) ...
> Setting up python3-tzlocal (4.2-1) ...
> Setting up python3-requests-oauthlib (1.3.0+ds-0.1) ...
> Setting up my-weather-indicator (0.10.1-0extras23.04.00) ...
> Processing triggers for fontconfig (2.13.1-4.4ubuntu1) ...
> Processing triggers for bamfdaemon (0.5.6+22.04.20220217-0ubuntu1) ...
> Rebuilding /usr/share/applications/bamf-2.index...
> Processing triggers for desktop-file-utils (0.26-1ubuntu4) ...
> Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
> Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
> tux@box:my-weather-indicator$ /opt/extras.ubuntu.com/my-weather-indicator/bin/my-weather-indicator --version
> Namespace GeocodeGlib not available for version 1.0
> Repository version required not present
> me@pro:my-weather-indicator$ 
> 
> **ps**, as mentioned, same behavior on 23.04. thanks for your time and help (-:
> **ps²:** al least the changelog under /opt/extras.ubuntu.com/my-weather-indicator/share/my-weather-indicator says: 
> 
> `my-weather-indicator (0.10.1-0extras23.04.00) jammy; `
> in line one, so that could be the version.
> **ps³**: an `grep -ir 'geocodeglib' my-weather-indicator` finds: 
> `my-weather-indicator/share/my-weather-indicator/myweatherindicator.py:    gi.require_version('GeocodeGlib', '1.0')`
> so let me see if this is manipulatable (-:
> 

Aaaaaargh!!!! what is that doing there? 
Thank you very much. I will update right now

**Comment by atareao** [comment]
Fixed on  my-weather-indicator - 0.10.2-0extras23.04.00
